unitTests/ui/CoreHelpersTest.ts

```diff
@@ -95,6 +96,14 @@ export function CoreHelperTest() {
       );
     });
 
+    it('highlightStreamText should not crash in case of duplicate terms on a text that starts with a word delimiter', () => {
+      const toHighlight = `'This is a bad text`;
```

Comment:
Why is the value of `toHighlight` considered bad text here? Is it because of the single quote at the start?

The test description also mentions "duplicate terms", but I don't see a duplicate in the test setup?

Reply:
Yes, because of the single quote; I'll add a comment to explain the corner case. The duplicate terms refer to this piece of code (same "aTerm" in both the "terms" and "phrases".

```
      const terms: IHighlightTerm = { aTerm: [] };
      const phrases: IHighlightPhrase = { aTerm: terms };
```